src/views/PlanTargets.vue

```diff
@@ -0,0 +1,201 @@
+            >{{s.name}}</option>
+            <option lat="" lon="" value="X">Custom Latitude and Longitude</option>
+          </b-select>
+        </b-field>
+        <div class="field has-addons">
+          <p class="control is-expanded">
+            <b-field label="Latitude">
+              <b-input type="text" id="lat1" v-model="user.lat1" required :disabled="user.localtime!='X'"/>
+            </b-field>
+          </p>
+          <p class="control is-expanded">
+            <b-field label="Longitude">
+              <b-input type="text" id="lon1" v-model="user.lon1" required :disabled="user.localtime!='X'"/>
+            </b-field>
+          </p>
+        </div>
+        <div class="field has-addons">
+          <p class="control">
+            <b-field label="Date/Time">
+              <b-datetimepicker 
+                id="dateobs" 
+                v-model="user.dateobs"
+                :timepicker="{ incrementMinutes:30 }"
+                :datetime-parser="(d) => {new Date(d)}"
+                required 
+                inline />
+            </b-field>
+          </p>
+        </div>
+      <b-field grouped>
+        <div v-if="user.localtime !== 'X'">
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="lcl" v-model="user.tzinfo">Local timezone</b-radio>
+        </div>
+        <div v-else>
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+        </div>
+      </b-field>
+        <div class="the-button">
+          <b-field class="buttons">
+            <b-button @click="submitForm" type="submit">Find Easy Targets</b-button>
+          </b-field>
+        </div>
+    </form>
+  </div>
+  <div class="results" id="results">
+    <div class="target-columns">
+      <div v-for="target in targlist" :target="target" :key="target.name">
+        <TargetCard :target="target" />
+      </div>
+    </div>
+  </div>
+</section>
+</template>
+
+<script>
+import Vue from 'vue';
+import axios from 'axios';
+import moment from 'moment';
+import TargetCard from '@/components/TargetCard.vue';
+import list from '../../public/data/easytargets.json';
+import helpers from '@/utils/helpers';
+
+export default {
+  name: 'PlanTargets',
+  components: {
+    TargetCard,
+  },
+  data() {
+    return {
+      site_info: {},
+      easylist: list,
+      target: {},
+      targlist: '',
+      user: {
+        lat1: '',
+        lon1: '',
+        dateobs: new Date(Math.round(new Date().getTime() / 1800000) * 1800000),
+        tzinfo: 'my',
+        offset: '',
+        localtime: '',
+        localoffset: '',
+      },
```

Comment:
Any reason this is in a parent 'user' object? I think it might be cleaner if all these parameters were nested in the same level under the 'data' return. Then we avoid the potential for the reactivity issue we found earlier. It's a minor thing though; if it works for now that's fine.

Reply:
It was mostly for my own debugging purposes - fixed though!